src/bank01_player.asm

```diff
@@ -780,12 +787,13 @@ scrollRoomMove:
     ld   A, $0f
     and  A, [HL]
     jr   NZ, .west_done_graphics
+    ; Queue up the next section of metatiles
     ld   A, [HL]
-    add  A, C
+    add  A, C ; adds $10 to queue the next section instead of the section actively being scrolled into
```

Comment:
In the first pass this is adding `$00`, correct? So the first call loads the first section and doesn't scroll, then the second call loads the second section and *does* scroll?

Reply:
Correct. I can add a comment to clarify that.

Reply:
Done